contracts/staking/RewardDelegators.sol

```diff
@@ -247,6 +249,31 @@ contract RewardDelegators is
             _aggregateReward = _aggregateReward + _reward;
         }
 
+        uint256 totalDelegations;
+        for (uint256 index = 0; index < _tokens.length; index++) {
+            bytes32 _token = _tokens[index];
+            
+            // assuming this is pond
+            if(_token == tokenList[0]){
+                totalDelegations += clusters[_cluster].totalDelegations[_token];
+            }
+            // assuming this is MPond
+            else if(_token == tokenList[1]){
+                totalDelegations += (1_000_000 * clusters[_cluster].totalDelegations[_token]);
+            }else{
+                revert("Resolve this issue");
```

Comment:
Make the revert message better, may be something on the lines of "invalid token for delegation". Also you can make this an assert instead of revert